pkg/envinfo/envinfo.go

```diff
@@ -205,6 +205,18 @@ func newProxyEnvInfo() proxyEnvInfo {
 func (esi *EnvSpecificInfo) SetConfiguration(parameter string, value interface{}) (err error) {
 	if parameter, ok := asLocallySupportedParameter(parameter); ok {
```

Comment:
Are the env values case sensitive? I can run `odo env set DebugPort`, but can't run `odo env set debugport`, and same for unset.

Reply:
Currently yes, but I can make it case insensitive.